src/spatial/models/heatmapModel.py

```
@@ -600,19 +599,32 @@
         Returns:
             The loaded model or None if loading fails.
         """
-        model_path = os.path.join(self.MODEL_DIR, f"{city_name}_rf_model.joblib")
+        model_path = os.path.join(config.MODEL_DIR, f"{city_name}_rf_model.joblib")
+        # Check for model locally first
         if os.path.exists(model_path):
             try:
                 model = joblib.load(model_path)
                 self.logger.info(f"Loaded model for '{city_name}' from {model_path}")
                 return model
             except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)